languages/csharp/config.json

```
@@ -24,6 +24,12 @@
         "concepts": ["booleans"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "chars",
+        "uuid": "e74c6cd6-add7-45d1-abcb-513418e9d2d0",
+        "concepts": ["chars", "StringBuilder"],
```

Concepts should use lowercase and kebab-case (see [this description](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts)):

```suggestion
        "concepts": ["chars", "string-builder"],
```